server/lib/schema/index.ts

```diff
@@ -2189,7 +2189,7 @@ const schema = new GraphQLSchema({
                     );
                   }
 
-                  tree.$add("oracle", user, transaction);
+                  await tree.$add("oracle", user, { transaction });
```

Comment:
I wonder if there was a race condition here previously that was causing the inconsistent behavior.

Reply:
Good catch on wrapping the transaction with an object.